dev/src/webpack/getDefaultWebpackConfig.ts

```diff
@@ -0,0 +1,288 @@
+                                //
+                                pattern: /path.join\((__dirname|__filename),.*'resources',.*'\)/ig,
+                                replacement: (match: any, _offset: any, _string: any): string => {
+                                    const pathExpression: string = match
+                                        .replace(/path\.join\((.*)\)/, '$1')
+                                        .replace(/\s*,\s*['"]/g, ` + '/`);
+                                    const requireExpression: string = `require(${pathExpression})`;
+                                    const resolvedExpression: string = `path.resolve(__dirname, ${requireExpression})`;
+                                    log('normal', `Rewrote resource reference: "${match}" => "${resolvedExpression}"`);
+                                    return resolvedExpression;
+                                }
+                            }
+                        ]
+                    })
+                },
+
+                {
+                    // This loader allows you to use a require() statement to get the path (not contents) to a loose file at runtime. Any file
+                    //   with the given extension referenced by a require() will be copied to the dist folder, and the require() at runtime will
+                    //   return a path to the copied file (not the contents).
+                    // For example:
+                    //   let myResourcePath = require(__dirname + '/resources/myresource.gif'); // (No, this will not work if not processed by webpack);
+                    //   (note that __dirname will not return the expected result at runtime because webpack flattens all source folders)
+                    // At pack time:
+                    //    <src>/<path>/<path>/resources/myresource.gif will be copied to dist/<path>/<path>/resources/myresource.gif
+                    // At runtime:
+                    //    require() will return the absolute path to dist/<path>/<path>/resources/myresource.gif
+                    test: /\.(png|jpg|gif|svg)$/,
+                    use: [
+                        {
+                            loader: 'file-loader',
+                            options: {
+                                name: (name: string): string => {
+                                    log('normal', `Extracting resource file ${name}`);
+                                    return '[path][name].[ext]';
+                                }
+                            }
+                        }
+                    ]
+                },
+
+                // Note: If you use`vscode-nls` to localize your extension than you likely also use`vscode-nls-dev` to create language bundles at build time.
+                // To support webpack, a loader has been added to vscode-nls-dev .Add the section below to the`modules/rules` configuration.
+                // {
```

Comment:
Can we just add this by default instead of commenting it out? All of our extensions should use vscode-nls eventually anyways

Reply:
As none are using it, it's not tested.

Reply:
And another dependence that wouldn't actually be used, so I'm hesitant to bake it in.

Reply:
functions and event grid use it :) I test it by selecting "Configure Display Language" and setting it to "pseudo". If you don't want to do this then I'm fine doing it myself since none of "your" extensions use it yet haha

Reply:
I'll enable it when we get to those, then.